sunpy/net/vso/vso.py

```diff
@@ -942,3 +944,45 @@ def _attrs_module(cls):
 
     def __del__(self):
         self.api.transport.session.close()
+
+    @classmethod
+    def register_values(cls):
+        try:
+            return {}  # cls.get_vso_values()
+        except Exception as e:
+            warnings.warn(f'Fetching VSO attrs failed with {e}')
+            return {}
+
+    def get_vso_values():
+        """
+        Reads the VSO keywords and returns a dict with them for the attrs we support.
+        """
```

Comment:
Can you add some kind of comment to this saying it's not being used and what it's designed to do?

Reply:
Fixed,